src/service/xhr-impl.js

```diff
@@ -409,6 +409,7 @@ export function assertSuccess(response) {
     if (response.status < 200 || response.status >= 300) {
       /** @const {!Error} */
       const err = user().createError(`HTTP error ${response.status}`);
+      err.response = response;
```

Comment:
I am, [here](https://github.com/ampproject/amphtml/pull/5920/files#diff-dec2ba5b2a0930ac31901ad01f85a069R154).

That makes me ask: did you see the diff of the `amp-access-laterpay.js` file? I saw someone complain about this on twitter the other day, but apparently when diffs are too big (I guess) it just does this:
<img width="999" alt="2016-12-09 at 14 29" src="https://cloud.githubusercontent.com/assets/38177/21050667/5520ea5a-be1c-11e6-90b5-a4de5d3752d4.png">
I was looking at the Files Changed tab and thinking "where did all of that code go? I wrote more code than this" and that's when I noticed this collapsed diff.